### PR TITLE
Add new conversation trigger, wire into FB, remove FB welcome setting

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: temba/channels/views.py [hunk 1]
```diff
@@ -1739,50 +1738,6 @@ def form_valid(self, form):
 
             return HttpResponseRedirect(reverse('channels.channel_configuration', args=[channel.id]))
 
-    class FacebookWelcome(ModalMixin, OrgPermsMixin, SmartUpdateView):
```
Comment (nicpottier) [inline temba/channels/views.py]: Don't need this view anymore, all done via Trigger

File: temba/triggers/models.py [hunk 2]
```diff
@@ -270,54 +318,28 @@ def find_flow_for_inbound_call(cls, contact):
...
+
         return [each_trigger.pk for each_trigger in triggers]
 
     @classmethod
     def apply_action_restore(cls, user, triggers):
-        m_last_triggered = triggers.filter(trigger_type=Trigger.TYPE_MISSED_CALL).order_by('-last_triggered', '-modified_on')
-        c_last_triggered = triggers.filter(trigger_type=Trigger.TYPE_CATCH_ALL).order_by('-last_triggered', '-modified_on')
-
-        remaining_triggers = triggers.exclude(pk__in=m_last_triggered).exclude(pk__in=c_last_triggered)
+        restore_priority = triggers.order_by('-last_triggered', '-modified_on')
```
Comment (nicpottier) [inline temba/triggers/models.py]: This was all refactored to be a lot more generic and readable.

Comment (ericnewcomer) [inline temba/triggers/models.py]: more readable indeed!

**Comment by ericnewcomer** [comment]
ICe!

**Pull request merged**